### ISSUE TITLE
[FEATURE-REQUEST] Shuffle data by default in vaex.ml.train_test_split 

### ISSUE THREAD

**Issue by Koushik667** [vaexio/vaex#1619]
**Description**
Please add support for shuffling of data by default while doing test train split in vaex ml, i was not aware of this until i saw the verbose comment.
I know that we could shuffle it manually before, but since sklearn's split function was doing it by default, thought this would also do the same. Is there any specific reason why this support is not there in vaex?
